packages/firestore/test/util/equality_matcher.ts

```diff
@@ -85,7 +85,7 @@ export function addEqualityMatcher() {
   afterEach(() => {
     if (originalFunction) {
       use(chai => {
-        const wrappedDefault = _super => {
+        return _super => {
```

Comment:
How confident are you that this is right?  It doesn't seem right to me...  Seems like this should be calling Assertion.overwriteMethod() to undo what beforeEach() did...  Maybe ping Josh or log a bug?